test/parallel/test-util-deprecate-invalid-code.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
Please use single quotes for strings throughout.

Reply:
Ah my local workspace had prettier enabled which I have disabled now. Updated PR.
Thanks !